### DISCUSSION TITLE
完成支付后无法使用

### DISCUSSION THREAD

**Discussion by cntceo** [getlantern/lantern-forum-cn#780] [General]
账号 469323024@qq.com 
支付完成，没有收到邮件，没有获得专业版
![WechatIMG502](https://github.com/user-attachments/assets/c01391f8-5132-41fd-b5aa-9d527f4a0c19)

**Comment by MaxMKMax** [comment]
@cntceo 好的. 我帮您提交问题. 谢谢您支持蓝灯!

**Reply by cntceo** [reply]
我还没有恢复啊

**Comment by cntceo** [comment]
我还没有恢复啊，能申请退款吗？
